modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,216 @@
++
+.Example of an NMState configuration for a customized `br-ex` bridge network
+[source,yaml]
+----
+interfaces:
+- name: enp2s0 <1>
+  type: ethernet <2>
+  state: up <3>
+  ipv4:
+    enabled: false <4>
+  ipv6:
+    enabled: false
+- name: br-ex
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
+  bridge:
+    port:
+    - name: enp2s0 <5>
+    - name: br-ex
+  - name: br-ex
+  type: ovs-interface
+  state: up
+  copy-mac-from: enp2s0
+  ipv4:
+    enabled: true
+    dhcp: true
+  ipv6:
+    enabled: false
+    dhcp: false
+# ...
+----
+<1> Name of the interface.
+<2> The type of interface. This example creates a bridge.
+<3> The requested state for the interface after creation.
+<4> Disables IPv4 and IPv6 in this example.
+<5> The node NIC to which the bridge attaches.
+
+. Use the `cat` command to base64-encode the contents of the NMState configuration:
++
+[source,terminal]
+----
+$ cat <nmstate_configuration>.yaml | base64 <1>
+----
+<1> Replace `<nmstate_configuration>` with the name of your NMState resource YAML file.
+
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration analogous to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker <2>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <3>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/cluster.yml
+# ...
+----
+<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. If you have a single global configuration specified in an `/etc/nmstate/openshift/cluster.yml` configuration file that you want to apply to all nodes in your cluster, you do not need to specify the hostname path for each node. The `worker` role is the default role for nodes in your cluster. Note that the `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
+<2> The name of the policy.
+<3> Writes the encoded base64 information to the specified path. 
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `NodeNetworkConfigurationPolicy (NNCP)` CR and define a customized `br-ex` bridge network configuration. Depending on your needs, ensure that you set a masquerade IP for either the `ipv4.address.ip`, `ipv6.address.ip`, or both parameters. A masquerade IP address must match an in-use IP address block.
```

Comment:
There are some Prerequisites here:
1. You used configure-ovs alternative to created br-ex bridge.
2. You installed Kubernetes NMState Operator.

Reply:
:heavy_check_mark:

Reply:
Hi @dfitzmau, the first Prerequisites is 'You used configure-ovs alternative to created br-ex bridge', not 'configure-ovs shell script'.

Reply:
Updated but I might be causing confusion here as the alternative to `configure-ovs` involves creating a customized br-ex bridge, so recreating it would not be required at day-2? The Day-2 activity should really centre around configuring an existing customized `br-ex` bridge?